Configs/.config/hyprdots/scripts/swwwallbash.sh

```diff
@@ -36,6 +36,7 @@ fi
 fn_wallbash () {
     local tplt="${1}"
     eval target=$(head -1 "${tplt}" | awk -F '|' '{print $1}')
+    [ ! -e ${target} ] && return 0
```

Comment:
cool thanks, but this will create more dependency as target file should pre-exist, so i think its better to check if the parent directory exist and not the wallbash file...
or we could use `touch "${target}" || return 0` condition instead...

Reply:
> We can't touch the file if parent does not exist I guess.

yup, touch would fail that's exactly why `||` is used to return 0...

Reply:
alright so ``` touch: cannot touch '/tmp/hjhj/hjhj': No such file or directory ```  as warning is okay right?

Reply:
hmm i think we can suppress this output and use `echo "[skip] $(dirname ${target})"` as you did before... so we get cleaner output